MAUI/Kanban-Board/Workflows.md

```diff
@@ -9,27 +9,356 @@ documentation: ug
+            Tags = new List<string> { "Feature", "Release" }
+        });
+
+        cardsDetails.Add(new KanbanModel()
+        {
+            ID = 13,
+            Title = "UWP - 13",
+            ImageURL = assemblyName + ".people_circle18.png",
+            Category = "Closed",
+            Description = "Fix cannot open user's default database SQL error.",
+            IndicatorFill = Colors.Purple,
+            Tags = new List<string> { "Bug", "Internal", "Release" }
+        });
+
+        cardsDetails.Add(new KanbanModel()
+        {
+            ID = 14,
+            Title = "Android - 14",
+            Category = "Closed",
+            ImageURL = assemblyName + ".people_circle19.png",
+            Description = "Arrange a web meeting with the customer to get the login page requirement.",
+            IndicatorFill = Colors.Red,
+            Tags = new List<string> { "Feature" }
+        });
+
+        cardsDetails.Add(new KanbanModel()
+        {
+            ID = 15,
+            Title = "Xamarin - 15",
+            Category = "Closed",
+            ImageURL = assemblyName + ".people_circle20.png",
+            Description = "Login page validation.",
+            IndicatorFill = Colors.Red,
+            Tags = new List<string> { "Bug" }
+        });
+
+        cardsDetails.Add(new KanbanModel()
+        {
+            ID = 16,
+            Title = "Xamarin - 16",
+            ImageURL = assemblyName + ".people_circle21.png",
+            Category = "Closed",
+            Description = "Test the application in the IE browser.",
+            IndicatorFill = Colors.Purple,
+            Tags = new List<string> { "Bug" }
+        });
+
+        cardsDetails.Add(new KanbanModel()
+        {
+            ID = 20,
+            Title = "UWP - 20",
+            ImageURL = assemblyName + ".people_circle22.png",
+            Category = "Closed",
+            Description = "Analyze stored procedure.",
+            IndicatorFill = Colors.Brown,
+            Tags = new List<string> { "CustomSample", "Customer", "Incident" }
+        });
+
+        cardsDetails.Add(new KanbanModel()
+        {
+            ID = 21,
+            Title = "Android - 21",
+            Category = "Closed",
+            ImageURL = assemblyName + ".people_circle23.png",
+            Description = "Arrange a web meeting with the customer to get editing requirements.",
+            IndicatorFill = Colors.Orange,
+            Tags = new List<string> { "Story", "Improvement" }
+        });
+
+        return cardsDetails;
+    }
+
+    #endregion
+}
 
 {% endhighlight %}
 
-By following the code snippet, you will notice that the card picked from the `Open` state is not allowed to be dropped in the `Code Review` or `Done` states because we have defined that the card can only move from the `Open` state to the `In Progress` state and not to any other states.
\ No newline at end of file
+{% endtabs %}
+
+![workflow-in-maui-kanban](images/workflow/workflow-in-maui-kanban.gif)
```

Comment:
@VishalOmprasad, replace gif, as it doesn't align with Material3 color design.

Reply:
Following with UX team to get relevant GIF image